src/components/ReportActionItemIOUAction.js

```diff
@@ -56,9 +47,7 @@ const ReportActionItemIOUAction = ({
                 shouldShowViewDetailsLink={Boolean(action.originalMessage.IOUReportID)}
                 onViewDetailsPressed={launchDetailsModal}
             />
-            {isMostRecentIOUReportAction
```

Comment:
I don't get why this is not needed anymore, wouldn't this mean that we would show the preview for a paid IOU? I thought we did not want that?

Reply:
Oh, is it because of [this](https://github.com/Expensify/Expensify.cash/pull/3404/files#diff-3597ac886dade8ed106d403fad7c534b6a0fd9423bf6081877749b0295fb3bc8R90)?

Reply:
Yeah exactly. That logic had to move to within the Preview, as once the `iouReport` is paid we cannot guarantee it will be stored locally -- and the iouReport will be null